Jenkinsfile

```diff
@@ -461,8 +513,24 @@ def dotnet(Closure body){
 }
 
 def dotnetWindows(Closure body){
-  def dockerTagName = 'docker.elastic.co/observability-ci/apm-agent-dotnet-windows:latest'
-  bat label: 'Docker Build', script: "docker build --tag ${dockerTagName}  -m 2GB .ci\\docker\\buildtools-windows"
+  def dockerUri = 'docker.elastic.co/';
+  def dockerTagName = "${dockerUri}observability-ci/apm-agent-dotnet-windows:latest"
+  dockerLogin(secret: "secret/apm-team/ci/docker-registry/prod",
+  registry: "docker.elastic.co")
+  // Remove this line once push/pull is enabled by Infra
+  dockerTagName = 'docker.io/elwpenn/windows-vstudio-msbuild:latest'
```

Comment:
push/pull for windows is enabled as far as I know

Reply:
doesn't work, nick lang has a fixed that should be deployed today